recipes/racon/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 
 build:
   number: 0
-  skip: True # [not py27]
+  skip: False
```

Comment:
Can you just delete the skip line now that is not used?

Reply:
Oh, yeah that would be the right thing to do. Thanks